views/me.hbs

```diff
@@ -43,42 +43,44 @@
 
             <a href="/dashboard/me/settings"><button class="settings">Settings</button></a>
             <a href="/logout"><button class="logout">Log Out</button></a>
-        </div>
+        </div> --}}
 
         <!-- APPLICATION STATUS -->
-        <div class='background theme-yellow container text-center'>
-            <h1>application status</h1>
+        <div class='background container text-center'>
+            <h1 class="app-title">application status</h1>
+            <hr class="line">
 
             <div class="status-container">
                 {{#if user.hasApplication}}
-                    <div class="status" style="background-color: {{statusColor}};">{{user.applicationStatus}}</div>
+                    <h2 class="description">Your application is under review!</h2>
+                    <div class="status pending">
+                        <h2 class="stats">{{user.applicationStatus}}</h3>
+                    </div>
                 {{else}}
```

Comment:
About the `if/else` stuff. Go take a look at the [`getMe()` function](https://github.com/ideahacks/ideahacks.la/blob/development/ideahacks/routes/dashboard/dashboard.js#L13) in the `routes` folder. You'll notice that the call to `res.render()` passes in some JSON. This JSON is what the Handlebars stuff can access: notice how it access the `user` object, as well as a the `statusColor` field.

If you take a look at how the HTML was constructed before, it distinguished between whether or not the user had an application. This is a useful distinction because we generally need to show more info if they don't have an application (before we had an "Apply" button, now we'll need an extra `<a>` tag).

If the user has an application, the only things that differ are the color, the text inside the colored box, and the application status description. These can all be either hard-coded into the backend and passed through that JSON or put into CSS classes to dynamically fill out this HTML when it loads.

One solution may look something like this:
```handlebars
{{#if user.hasApplication}}
<h2 class="description">{{statusDescription}}</h2>
<div class="status {{user.applicationStatus}}">
    <h2 class="stats">{{user.applicationStatus}}</h2>
</div>
{{else}}
<h2 class="description"><a href="/dashboard/application">Click here to apply now!</a></h2>
<div class="status no-application">
    <h2 class="stats">No Application</h2>
</div>
{{/if}}
```

Notice how this solution depends on the backend to tell it what the status description should be and uses a CSS class of the same name as the application status to do the background and text coloring. It's still useful to distinguish from people with no application because the HTML is structured slightly differently and `user.applicationStatus` doesn't have a value for no application. Of course, the backend doesn't handle this approach, but with some modifications to the `getMe()` function I mentioned earlier, you should be able to get it to work. One thing to pay attention to is that you'll need to shut down and restart your `npm start` every time you make a change to the backend to see the changes. Feel free to DM me if you need more explanation